### PR TITLE
Added code to check and delete cookie.

### PR DESCRIPTION
**Pull request by omnishu**
Although code is not efficient.
But solves the purpose for now.

### CODE REVIEW

**Comment by Noitidart** [comment]
great work ill review when i get some time

File: bootstrap.js [hunk 1]
```diff
@@ -250,4 +252,20 @@ function xhr(aStr, aOptions={}) {
...
 	return deferredMain_xhr.promise;
 }
-// end - common helper functions
\ No newline at end of file
+// end - common helper functions
+
+function getCookie(){
+	if(Components.classes["@mozilla.org/cookiemanager;1"]
+		.getService(Components.interfaces.nsICookieManager2)
+		.cookieExists({
```
Comment (Noitidart) [inline bootstrap.js]: `Services.cookies.cookieExists` didnt work here? Its more perofmance to use Services module.

File: bootstrap.js [hunk 2]
```diff
@@ -250,4 +252,20 @@ function xhr(aStr, aOptions={}) {
...
+	      host: ".example.com",
+	      path: "/",
+	      name: "cookieName",
+	})){
+		doPost("Cookie Found");
+		Components.classes["@mozilla.org/cookiemanager;1"].getService(Components.interfaces.nsICookieManager).remove(".example.com","cookieName","/",false);
```
Comment (Noitidart) [inline bootstrap.js]: `Services.cookies.remove` didnt work here?

**Comment by omnishu** [comment]
Updated the PR. Please check if this is awesome :)

**Comment by Noitidart** [comment]
Looks fantastic! Great job! Merging pr :)

**Pull request merged**